pycobertura/utils.py

```diff
@@ -208,3 +210,15 @@ def hunkify_lines(lines, context=3):
         hunks.append(hunk)
 
     return hunks
+
+
+def get_output(command, working_folder=None):
+    logging.debug("Executing %s in %s", command, working_folder)
+
+    try:
+        output = subprocess.check_output(shlex.split(command), cwd=working_folder)
+        return output.decode("utf-8")
+    except OSError:
+        logging.error("Command being executed: {}".format(command))
+        raise
+
```

Comment:
I have noticed that in both my pull requests I left trailing whitespaces.

Would you mind if I use black to format sources? I gave it a try on your code (that is very nicely formatted), and if you were OK to switch to a line length of 88 characters, it would, I think, fit your needs.

Reply:
I've never used black before. Also the homepage doesn't make it clear to what it covers. It seems like it would work like "prettier", no? Does it work only for Python?

Reply:
If we were to use Black, I'd rather do this in a separate PR if you don't mind.